chall11/vde-dios.c

```diff
@@ -0,0 +1,54 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <errno.h>
+#include <ctype.h>
+
+char *ft_goto_parser(const char *code)
+{
+    char *next;
+    char *word;
+    char *aux = NULL;
+    char *output = NULL;
+    char *tmp = NULL;
+    long n_line = 0;
+    char *input = strdup(code);
+
+    next = strtok_r(input, "\n", &input);
+    while (next)
+    {
+        if (!isdigit(*next))
+        {
+            aux = next;
+            while(!isdigit(*aux))
+                aux++;
+            next = strtok_r(input, "\n", &input);
+            while(!strstr(next, aux))
+            {
+                next = strtok_r(input, "\n", &input);
+                if (!next)
+                    return (strdup("Infinite loop !"));
```

Comment:
Not the best way to detect infinite loops.

For example you detect infinite loops if there's a line like this: `"goto  10  \n"` with more than one space after the line number.

Reply:
Ooops.. in this challenge I just tried the examples you provided (and similar ones) worked. Never though that was asked but makes sense  🤷‍♂️